.github/pull_request_template.md

```diff
@@ -17,12 +17,6 @@ PR Metadata
 ### Description
 <!--- Describe your changes in detail -->
 
-### Related GitHub [Issues](../issues) and/or ClickUp Task(s)
```

Comment:
How to deal with [this use case](https://github.com/DeepX-inc/caisson_ros_pkg/pull/176)?
Sometimes we still want to link/mention other PRs in this or other repos.

Reply:
I don't think we need a dedicated section in the template for such cases.
How about mentioning that as an example of what should go into the metadata?

Reply:
Okay. Sounds good.

Reply:
How about now?

Reply:
Looks good.